AK/Format.h

```diff
@@ -407,14 +407,14 @@ struct Formatter<const char*> : Formatter<StringView> {
     }
 };
 template<>
-struct Formatter<char*> : Formatter<const char*> {
+struct Formatter<char*> : Formatter<char const*> {
 };
 template<size_t Size>
-struct Formatter<char[Size]> : Formatter<const char*> {
-};
+struct Formatter<char[Size]> : Formatter<char const*> {};
+// clang-format off
```

Comment:
This will be fixed in 14.0.0:
https://github.com/llvm/llvm-project/issues/53567

Reply:
Yeah, waiting until 14.0.0 is probably the best bet, I didn't know this was still an rc